Makefile

```diff
@@ -1,28 +1,25 @@
-all: vendor
+all:
 	go build -o cni-noop ./cmd/cni-noop
 	go build -o cnispawn ./cmd/cnispawn
 	go build -o kubeadm-nspawn ./cmd/kubeadm-nspawn
 
-glide:
-	@which glide || go get -u github.com/Masterminds/glide
-
+vendor: glide.lock | glide
+	glide --quiet install --strip-vendor
 glide.lock: glide.yaml | glide
```

Comment:
I am unable to reproduce the error I had, so I am not sure if the explanation is correct...

@schu what do you think of changing the `|` as I suggested above?